_spinnaker/operator-config.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: Spinnaker Operator Reference
+order: 12
+---
+
+This document describes available fields in the SpinnakerService CRD used by Spinnaker Operator, as well as a description of the ConfigMap where Spinnaker stores its configuration.
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+# SpinnakerService CRD
+The following example shows the general structure of a manifest file for SpinnakerService:
+
+    apiVersion: spinnaker.io/v1alpha1
```

Comment:
Should it be wrapped in a  
"```yaml
...
```
"?

It seems to be rendered correctly in GitHub's interface. The YAML tag should do syntax highlighting.

Reply:
^ meant "three backticks followed by "yaml", etc."